README.md

```diff
@@ -2,7 +2,10 @@
 
 # `posteriordb-python`: a python library to work with `posteriordb`
 
-This repository contain the python library to easily work with the [posteriordb](https://github.com/stan-dev/posteriordb) repository. The library included database contain convenience functions to access data, model code and information for individual posteriors, models, data and reference draws.
+This repository contains the R package to efficiently work with the [posteriordb](https://github.com/stan-dev/posteriordb) repository. The R package includes convenience functions to access data, model code and information for individual posteriors, models, data and draws.
```

Comment:
I'm confused---under the posteriordb-python heading it now says it contains an R package.  Am I missing something?